packages/pwa/package.json

```diff
@@ -188,26 +188,16 @@
         ],
         "defaultCurrency": "GBP",
         "supportedLocales": [
-            {
-                "id": "en-GB",
-                "preferredCurrency": "GBP"
-            },
-            {
-                "id": "fr-FR",
-                "preferredCurrency": "EUR"
-            },
-            {
-                "id": "it-IT",
-                "preferredCurrency": "EUR"
-            },
-            {
-                "id": "zh-CN",
-                "preferredCurrency": "CNY"
-            },
-            {
-                "id": "ja-JP",
-                "preferredCurrency": "JPY"
-            }
+            { "id": "de-DE", "preferredCurrency": "EUR" },
+            { "id": "en-GB", "preferredCurrency": "GBP" },
+            { "id": "es-MX", "preferredCurrency": "MXN" },
+            { "id": "fr-FR", "preferredCurrency": "EUR" },
+            { "id": "it-IT", "preferredCurrency": "EUR" },
+            { "id": "ja-JP", "preferredCurrency": "JPY" },
+            { "id": "ko-KR", "preferredCurrency": "KRW" },
+            { "id": "pt-BR", "preferredCurrency": "BRL" },
```

Comment:
There are new languages/locales that were not there before. To be able to show them properly on the website, I had to add new locales and currencies via Business Manager (BM). Thankfully, it was quite straightforward to do. I can show you if you want.

And yes, we still need to back up the site configs in BM. I'll make sure to back them up.

Reply:
Also, do you want to add these notes into the readme file?

Reply:
@alexvuong Ok I've updated the translations readme:
- I saw that Vincent C had already written about something similar. So I just built on top of it.
- I took some time to clean up the readme too.